spec.bs

```diff
@@ -2266,6 +2267,20 @@ The network revocation mechanism requires the following monkeypatches to the [[F
   (<a href="https://github.com/WICG/fenced-frame/issues/191">WICG/fenced-frame#191</a>)
 </div>
 
+<div algorithm=resolve-an-origin-patch>
+  Modify the [=resolve an origin=] algorithm to include an extra parameter: a [=URL=] |url|.
+  
+  Insert these steps before step 1:
+  
+  1. Let |nonce| be <var ignore>key</var>'s [=network partition key/nonce=].
+  
+  1. If the user agent's [=network revocation exemption map=][|nonce|] [=map/exists=], but [=network
+     revocation exemption map=][|nonce|][|url|] does not [=map/exist=], then:
+     
+    1. If the user agent's [=network revocation nonce set=] [=set/contains=] |nonce|, then return
```

Comment:
```suggestion
     1. If the user agent's [=network revocation nonce set=] [=set/contains=] |nonce|, then return
```
Add 1 extra space so the `1.` lines up with the start of the text on the line above.